src/custom-types.json

```diff
@@ -637,6 +637,11 @@
   "image/x-xcf": {
     "compressible": false
   },
+  "image/vnd.ms-dds" : {
+    "sources": ["https://docs.microsoft.com/en-us/windows/win32/wic/dds-format-overview"],
+    "extensions: ["dds"],
```

Comment:
It looks like the tests are failing because there is a missing quote on this line, making the JSON file no longer parsable.

Reply:
I also caught that, and its fixed, thank you.